sda/cmd/ingest/ingest.go

```diff
@@ -257,8 +258,14 @@ func main() {
 				}
 
 				file, err := inbox.NewFileReader(message.FilePath)
-				if err != nil {
+				if err != nil { //nolint:nestif
 					log.Errorf("Failed to open file to ingest reason: (%s)", err.Error())
+					if strings.Contains(err.Error(), "no such file or directory") || strings.Contains(err.Error(), "NoSuchKey:") {
```

Comment:
I'm not sure I understand the thinking why we test for specific errors here.

To me, we're in a branch that will send to the `error` queue for informing/allowing manual interaction. I believe those errors all make sense to add to the file event log - the exception would if there's confidentiality reasons, but I don't see any such.

Reply:
S3 and NFS can have newtork hickups and in those cases we don't want to stop the ingestion and report an error, we just try again a bit later.

Reply:
That makes sense, but shouldn't that be the same for e.g. whatever to send to the `error` queue?

Reply:
There needs to be some refactoring later on, because if we `nack` a message we should not send it to the error queue.

I didn't want to include those changes here in order to keep the PR contained in scope.